.github/workflows/syncPluginApplyExample.yml

```diff
@@ -0,0 +1,30 @@
+name: Synchronize plugin apply example
+
+on:
+  pull_request:
+    branches:
+    - master
+    types:
+    - closed
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Check repository
+      run: |
+        if [ "${{ github.event.pull_request.base.repo.html_url }}" != "${{ github.event.pull_request.head.repo.html_url }}" ]; then
+          echo "ERROR: Cross-repository pull requests, e.g. from forks, are not supported."
+          exit 1
+        fi
+    - name: Create sync pull request
```

Comment:
One step is marked with _- uses_ and then _- name_. The other with _- name_ and then _uses_. Is there a convention on this or some meaning?

Reply:
`name` field is not mandatory, so when omitted a step could contain just `- uses`. I added name to the checkout step to make it look more readable.